docs/introduction/faq.md

```diff
@@ -0,0 +1,109 @@
+Every phing target is a task that can be executed simply by php phing <target-name> command.
+See more about phing targets in [Phing Targets](./phing-targets.md).
+
+## What are the data fixtures good for?
+Data fixtures are actually demo data that are already available in the Shopys Framework.
+There are two kinds of demo data, the singledomain and the multidomain.
+For the installation of the basic demo data, use the phing target `db-fixtures-demo-singledomain`.
+For the installation of the multidomain demo data, use the phing target `db-fixtures-demo-multidomain`.
+These phing targets are also triggered as the part of others phing targets, see `build.xml` and `build-dev.xml`.
+Demo data are used, for example, during the automatic tests or if you want to install Shopys Framework with some basic data.
+Do not forget to extend the demo data when implementing some new features.
+
+## How to perform a change of domain URL?
+The change of domain url requires two steps.
+In the first step, you need to modify the domain url in the configuration file `app/config/domains.yml`.
+In the second step, you need to replace all occurrences of the old url address in the database with the new url address.
+This scenario is described in more detail in the tutorial [How to Set Up Domains and Locales (Languages)](./how-to-set-up-domains-and-locales.md#4-change-the-url-address-for-an-existing-domain).
+
+## How to use migrations and why the developers should use shopsys:migrations:generate instead of the default Doctrine one?
+Migrations (or also sometimes calld as database migrations) are used to unify the database schema with ORM.
+On Shopsys Framework, you can use the phing target `db-migrations-generate` for migrations generation.
+Compared to the standard migrations generation process from Doctrine, this phing target does not generate "irreversible" migrations, such as migrations with the operations `DROP` or `DELETE`.
+Migrations are described more in detail in the docs [Database Migrations](./database-migrations.md)
+
+## What are the diff versions of coding standards check commands and what are their limits?
+Some of the coding standards check commands are available in two forms.
+The first basic form is used to check all files.
+The second additional form, commands with the suffix `-diff`, is used to check only modified files.
+For example, the phing target `standards` starts checking of all files in the application while the phing target `standards-diff` starts checking only the modified files.
+Modifications are detected via git by comparison against the origin/master version.
+
+## Is the application https ready or does it need some extra setting?
+Shopys Framework is fully prepared for HTTPS.
+
+## What about translations and language constants export and import?
+To set up the user translations of labels and messages, use the files `messages.cs.po` and `validators.cs.po`, where `cs` represents the version of translations for a locale `cs`.
```

Comment:
Btw: maybe it would be better to use en, so it is more clear for none czech users.
- done

where cs represents the locale.
- done

To set up the user translations of labels and messages use the files ...
- so I can use this files for user translations of products, categories, demo data, too?

Reply:
You can use it to translate almost anything you think of. Of course, it would be weird to use it for products translations when there is already another better solution.